charts/cryostat/templates/deployment.yaml

```diff
@@ -190,6 +190,23 @@ spec:
           securityContext:
             {{- toYaml .Values.storage.securityContext | nindent 12 }}
         {{- if not .Values.minimal }}
+        - name: {{ printf "%s-%s" .Chart.Name "oauth2proxy" }}
+          securityContext:
+            {{- toYaml .Values.oauth2Proxy.securityContext | nindent 12 }}
+          image: "{{ .Values.oauth2Proxy.image.repository }}:{{ .Values.oauth2Proxy.image.tag }}"
+          imagePullPolicy: {{ .Values.oauth2Proxy.image.pullPolicy }}
+          env:
+            - name: redirectURL
+              value: "http://localhost:8181/oauth2/callback"
```

Comment:
Should this be port 8181? I would expect it to be 8080 as the default for this container, since 8181 is Cryostat's HTTP port.

Reply:
right..